include/enums/cpp_tensor_type.h

```diff
@@ -87,29 +87,29 @@ inline TensorType convert_tensor_type(CTensorType type) {
 //! Helper method to convert between CTensorType and TensorType
 inline CTensorType convert_tensor_type(TensorType type) {
     CTensorType t;
-    switch(type) {
-        case TensorType::dbl :
+    switch (type) {
```

Comment:
We should probably add the ``invalid`` type to these switch cases too.

Reply:
done